<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registrarse</class>
 <widget class="QDialog" name="Registrarse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>485</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Droid Sans</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Mi Libreria</string>
  </property>
  <property name="windowOpacity">
   <double>0.970000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #333;
color: #f1f1f1;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="registrarmeLabel">
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Registrarme</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="usuarioLabel">
     <property name="text">
      <string>Usuario</string>
     </property>
     <property name="buddy">
      <cstring>usuarioInputNuevo</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="usuarioInputNuevo">
     <property name="styleSheet">
      <string notr="true">background-color: #535353;
color:#f1f1f1;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Usuario</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="nombreLabel">
     <property name="text">
      <string>Nombre Completo</string>
     </property>
     <property name="buddy">
      <cstring>usuarioInputNuevo</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="nombreInputNuevo">
     <property name="styleSheet">
      <string notr="true">background-color: #535353;
color:#f1f1f1;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Nombre Completo</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="fechaLabel">
     <property name="text">
      <string>Fecha Nacimiento</string>
     </property>
     <property name="buddy">
      <cstring>usuarioInputNuevo</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDateEdit" name="fechaInputNuevo">
     <property name="styleSheet">
      <string notr="true">background-color: #535353;
color: #f1f1f1;</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhPreferNumbers</set>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="displayFormat">
      <string>dd/MM/yyyy</string>
     </property>
     <property name="date">
      <date>
       <year>2018</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="contrasenaLabel">
     <property name="text">
      <string>Contraseña</string>
     </property>
     <property name="buddy">
      <cstring>usuarioInputNuevo</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="contrasenaInputNuevo">
     <property name="styleSheet">
      <string notr="true">background-color: #535353;
color:#f1f1f1;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::PasswordEchoOnEdit</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Contraseña</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="correoLabel">
     <property name="text">
      <string>Correo</string>
     </property>
     <property name="buddy">
      <cstring>usuarioInputNuevo</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="correoInputNuevo">
     <property name="styleSheet">
      <string notr="true">background-color: #535353;
color:#f1f1f1;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Correo</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="registrarmeBotonNuevo">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>51</red>
           <green>51</green>
           <blue>51</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>153</red>
           <green>153</green>
           <blue>153</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Dark">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>231</red>
           <green>231</green>
           <blue>231</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>51</red>
           <green>51</green>
           <blue>51</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>51</red>
           <green>51</green>
           <blue>51</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>153</red>
           <green>153</green>
           <blue>153</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>153</red>
           <green>153</green>
           <blue>153</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>51</red>
           <green>51</green>
           <blue>51</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>153</red>
           <green>153</green>
           <blue>153</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Dark">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>231</red>
           <green>231</green>
           <blue>231</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>51</red>
           <green>51</green>
           <blue>51</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>51</red>
           <green>51</green>
           <blue>51</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>153</red>
           <green>153</green>
           <blue>153</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>153</red>
           <green>153</green>
           <blue>153</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>51</red>
           <green>51</green>
           <blue>51</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>153</red>
           <green>153</green>
           <blue>153</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Dark">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>231</red>
           <green>231</green>
           <blue>231</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>51</red>
           <green>51</green>
           <blue>51</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>51</red>
           <green>51</green>
           <blue>51</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>153</red>
           <green>153</green>
           <blue>153</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>153</red>
           <green>153</green>
           <blue>153</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <family>Droid Sans</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#999;
color:#333;</string>
     </property>
     <property name="text">
      <string>Registrarme</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Ya tienes cuenta?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="iniciarBotonNuevo">
     <property name="font">
      <font>
       <family>Droid Sans</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#999;
color:#333;</string>
     </property>
     <property name="text">
      <string>Iniciar Sesión</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
